### DISCUSSION TITLE
Güllezubringen direkt zum Ausbringer

### DISCUSSION THREAD

**Discussion by Markowend1** [Courseplay/Courseplay_FS22#3304] [Ideas]
Ein Gülleausbringer (zb. Holmer mit Scheibenegge) bringt Gülle auf einem Feld aus. Ist er leer, bleibt er stehen. Nun würde ich mir wünschen, dass der Autodrivezubringer am Feldrand wartet bis der Ausbringer fast leer ist und dann automatisch zum Ausbringer in den Trigger fährt. Jetzt kann der Ausbringer sich auftanken und weiter arbeiten. Der leere Zubringer fährt vom Feld und holt neue Gülle. 

Falls das hier falsch adressiert ist tut es mir leid. Habe gerade frisch den Umzug von FS17 zu FS22 gewagt.

**Comment by Tensuko** [comment]
Steht schon unter Issues.
